WordPress/Classes/ViewRelated/Jetpack/Jetpack Restore/Restore Options/JetpackRestoreOptionsViewController.swift

```diff
@@ -46,4 +56,20 @@ class JetpackRestoreOptionsViewController: BaseRestoreOptionsViewController {
         self.navigationController?.pushViewController(warningVC, animated: true)
     }
 
+    override func detailActionButtonTapped() {
+        guard let jetpackSettingsURL = URL(string: "https://wordpress.com/settings/jetpack/\(site.displayURL)") else {
```

Comment:
When using my testing site that includes a `/` in the URL the WebView ends up white and doesn't load. We can avoid this by passing the site ID instead of the URL. 

```suggestion
        guard let jetpackSettingsURL = URL(string: "https://wordpress.com/settings/jetpack/\(site.siteID)") else {
```

Reply:
Fixed! d1057a0